apis

```diff
@@ -0,0 +1,75 @@
+User creation API:
+
+/users
+  Method: POST
+  Content (application-json): {"email":"<something>", "name":"<something>", "password":"<something>"}
```

Comment:
for name - we want first and last yeah? so:

```{"email":"<something>", "first_name":"<something>", "last_name":"<something>", "password":"<something>"}

Reply:
Sure sounds good.